scss/modules/_forms.scss

```diff
@@ -58,57 +90,6 @@
     }
   }
 
-  input:focus,
-  textarea:focus {
-    border-color: $brand-color;
```

Comment:
@anthonydillon If you look at line 119 you should see the ```:active / :focus``` should be controlled by the ```%input-elements``` placeholder, felt this was a more efficient way to control it.